vars/parseStageInfo.groovy

```diff
@@ -114,22 +114,27 @@ def call(Map config = [:]) {
     result['node_count'] = 1
 
     if (env.STAGE_NAME.contains('Functional')) {
+      if (env.BRANCH_NAME == 'master') {
+          core_tag = 'pr'
+      } else {
+          core_tag = 'pre'
```

Comment:
The tag for non-master (PRs, but actually any other [release] branch) now `pre`?  What does `pre` mean?

Reply:
I put this in one of the comments, the tags should be "pr", and "landing" but in order to test that I'd need to do a search-and-replace across all tests, so for now I'm using "pre" for tests that run on PRs, and "pr" for tests that run on master, just to prove the concept with making minimal changes to the src.